crates/test-utils/input/square.cairo

```diff
@@ -0,0 +1,7 @@
+fn square(a: i32) -> i32 {
+  a*a
+}
+
+fn main() -> i32 {
+  return square(3);
+}
```

Comment:
This should not live in the utils crate. The main E2E tests should probably live in "driver", and so should the test inputs.